src/ufuncs.c

```diff
@@ -42,7 +49,6 @@ static void *is_empty_data[1] = {GEOSisEmpty_r};
 static char GEOSisSimpleAllTypes_r(void *context, void *geom) {
     int type = GEOSGeomTypeId_r(context, geom);
     if (type == -1) {
-        RAISE_ILLEGAL_GEOS;
```

Comment:
maybe add a comment here that "2" is used to catch the error in `Y_b_func` below (as my first reaction was "don't we need to set an error?")